ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -71,6 +72,21 @@ <h3>Source database details:</h3>
                         </button></span>
                 </p>
             </div>
+            <div *ngIf="isSharded && (selectedMigrationType === 'bulk')">
+                <h3>Source databases details:</h3>
+                <p class="point">
+                    <span class="bullet">1</span>
+                    <span>Setup Source databases details</span><span><button mat-button class="configure"
+                            (click)="openShardedBulkSourceDetailsForm()" [disabled]="isMigrationInProgress">
+                            Configure
+                            <mat-icon>edit</mat-icon>
+                            <mat-icon *ngIf="isSourceDetailsSet" class="success" matTooltip="Source details configured"
```

Comment:
The form validates the connection at the time of submitting and the form submission would fail if the connection is unsuccessful. 
I agree though, having a test connection button is a better UI experience.